.github/workflows/tests.yml

```diff
@@ -20,9 +20,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        typo3: [ ^10.4 ]
+        typo3: [ ^10.4, ^11.5 ]
         php: [ '7.2', '7.3', '7.4' ]
         experimental: [false]
+        include:
+          - typo3: ^11.5
+            php: '8.0'
```

Comment:
You can include PHP `8.1` too.

Reply:
Added 8.1 to test matrix. Also in TYPO3_Restler Pull-Request.